Description: mdcheck: Log when done
 Currently mdcheck (when called with `--duration`) logs only the
 beginning of the check, the pausing and the continuation but not the
 completion.

 So, log the completion, too, so that it can be determined how long the
 raid check took.

 2020-05-08T18:00:02+02:00 deadpool root: mdcheck start checking /dev/md0
 2020-05-08T18:00:02+02:00 deadpool root: mdcheck start checking /dev/md1
 2020-05-09T15:32:04+02:00 deadpool root: mdcheck finished checking /dev/md1
 2020-05-09T17:38:04+02:00 deadpool root: mdcheck finished checking /dev/md0

Author: Donald Buczek <buczek@molgen.mpg.de>
Origin: upstream, https://git.kernel.org/pub/scm/utils/mdadm/mdadm.git/commit/?id=3b7aae927bdb6e150d1aaf3aaf0d183a06abda0b
Bug-Ubuntu: https://launchpad.net/bugs/1852747
--- a/misc/mdcheck
+++ b/misc/mdcheck
@@ -125,11 +125,13 @@
 	do
 		eval fl=\$MD_${i}_fl
 		eval sys=\$MD_${i}_sys
+		eval dev=\$MD_${i}_dev
 
 		if [ -z "$fl" ]; then continue; fi
 
 		if [ "`cat $sys/md/sync_action`" != 'check' ]
 		then
+			logger -p daemon.info mdcheck finished checking $dev
 			eval MD_${i}_fl=
 			rm -f $fl
 			continue;
